packages/story-editor/src/components/panels/document/taxonomies/taxonomies.js

```diff
@@ -35,19 +37,33 @@ function TaxonomiesPanel(props) {
     return null;
```

Comment:
You'd need to use `sortedTaxonomies
        .filter((taxonomy) => ENABLED_TAXONOMIES.includes(taxonomy.rest_base))` here as well in theory.